hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -1247,9 +1254,10 @@ private Bytes startBlockHashFrom(@NonNull final State state) {
                 .stateChanges(new StateChanges(blockStreamInfo.blockEndTime(), List.of(blockStreamInfoChange)))
                 .build();
         // So we can combine this last leaf's has with the size and rightmost hashes
-        // store from the pending output tree to recompute its final root hash
+        // store from the pending state changes tree to recompute its final root hash
         final var penultimateOutputTreeStatus = new StreamingTreeHasher.Status(
```

Comment:
_Nit_ - could update this name to `penultimateStateChangesTreeStatus` in later PR.

Reply:
Changed. Thanks Michael